packages/react/src/components/CTASection/__tests__/CTASection.test.js

```diff
@@ -10,33 +10,45 @@ import CTASection from '../CTASection';
 import { mount } from 'enzyme';
 import React from 'react';
 
-const contentItemsProps = [
-  {
-    heading: 'Get connected',
-    copy: `
-      IBM DevOps partners have a wide range of expertise.
-      Find one to build the right solution for you.
-      `,
-    cta: {
-      copy: 'Find a partner',
-      type: 'external',
-      href: 'https://example.com/',
+const contentItemsProps = {
+  heading: 'Content item heading',
```

Comment:
not a big deal, but, actually, both `heading` and `copy` belongs to `ContentBlock` and not `ContentItem`.